packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewHostApiImpl.java

```diff
@@ -104,6 +108,35 @@ public View getView() {
     @Override
     public void dispose() {}
 
+    // TODO(bparrishMines): This should be removed once https://github.com/flutter/engine/pull/40771 makes it to stable.
+    // Temporary fix for https://github.com/flutter/flutter/issues/92165. The FlutterView is setting
+    // setImportantForAutofill(IMPORTANT_FOR_AUTOFILL_YES_EXCLUDE_DESCENDANTS) which prevents this
+    // view from automatically being traversed for autofill.
+    @Override
+    protected void onFocusChanged(boolean focused, int direction, Rect previouslyFocusedRect) {
+      super.onFocusChanged(focused, direction, previouslyFocusedRect);
+      if (focused && Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+        final FlutterView flutterView = tryFindFlutterView();
+        if (flutterView != null) {
+          flutterView.setImportantForAutofill(IMPORTANT_FOR_AUTOFILL_YES);
+        }
+      }
+    }
+
+    // Attempt to traverse the parents of this view until a FlutterView is found.
+    private FlutterView tryFindFlutterView() {
+      ViewParent currentView = this;
+
+      while (currentView.getParent() != null) {
+        currentView = currentView.getParent();
+        if (currentView instanceof FlutterView) {
+          return (FlutterView) currentView;
+        }
+      }
+
+      return null;
+    }
```

Comment:
The parents structure of Hybrid Composition and TLHC are different, so this should handle both cases.